apps/dashboard/app/models/user_configuration.rb

```diff
@@ -0,0 +1,90 @@
+class UserConfiguration
+
+  USER_PROPERTIES = [
+    ConfigurationProperty.property(name: :dashboard_header_img_logo, read_from_env: true),
+    # Whether we display the Dashboard logo image
+    ConfigurationProperty.with_boolean_mapper(name: :disable_dashboard_logo, default_value: false, read_from_env: true, env_names: ['OOD_DISABLE_DASHBOARD_LOGO', 'DISABLE_DASHBOARD_LOGO']),
```

Comment:
`disable_dashboard_logo`  has been configured to prefer `OOD_DISABLE_DASHBOARD_LOGO` but will support both, old and new.

Reply:
that works! fits nicely in the implementation too. I'm good with this.